core/storage/user/gae_models_test.py

```diff
@@ -777,6 +955,22 @@ def test_get_deletion_policy(self) -> None:
             user_models.UserContributionsModel.get_deletion_policy(),
             base_models.DELETION_POLICY.DELETE)
 
+    def test_get_model_association_to_user(self) -> None:
+        self.assertEqual(
+            user_models.UserContributionsModel.get_model_association_to_user(),
+            base_models.MODEL_ASSOCIATION_TO_USER.ONE_INSTANCE_PER_USER)
+
+    def test_get_export_policy(self) -> None:
+        self.assertEqual(
+            user_models.UserContributionsModel.get_export_policy(),
+        {'created_on': base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
I would suggest breaking after '{' and indenting the subsequent 'created_on' line by 4.